packages/core/core/src/parcel-v3/plugins/Resolver.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export class ResolverNapi {
```

Comment:
don't we have an existing resolver interface?

Reply:
I'll probably rename this, it's more like a `ResolverController` that will be used to orchestrate calling resolvers from the Rust core